test-e2e/project-crud.js

```diff
@@ -102,6 +103,7 @@ test('CRUD operations', async (t) => {
       const project = await manager.getProject(projectId)
       /** @type {any[]} */
       const updates = []
+      /** @type {Promise<MapeoDoc>[]} */
```

Comment:
I don't know why this was required as part of the upgrade, but it was easy to fix.